templates/etc/httpd/conf.d/_ssl_username.conf.erb

```diff
@@ -0,0 +1,5 @@
+#
+# Apache configuration file for Pulp's Puppet support
```

Comment:
This seems incorrect documentation, I think you'd  be OK reducing this to a one line comment above the directive.